src/styles/blocks/stars.css

```diff
@@ -0,0 +1,19 @@
+.rate-stars {
+  display: flex;
+}
+
+.rate-stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.rate-stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.rate-stars__star--active {
+  background-image: url(../../images/star-active.svg);
+}
```

Comment:
addition to my comment in html file, i think that you don’t need to hardcode this

Reply:
yeap, you need to re-use the same styles that you already wrote in the homework with stars